integrations/deepeval.md

```diff
@@ -0,0 +1,93 @@
+---
+layout: integration
+name: DeepEval
+description: Use the DeepEval evaluation framework to calculate model-based metrics 
+authors:
+    - name: deepset
+      socials:
+        github: deepset-ai
+        twitter: deepset_ai
+        linkedin: deepset-ai
+pypi: https://pypi.org/project/deepeval-haystack
+repo: https://github.com/deepset-ai/haystack-core-integrations/tree/main/integrations/deepeval
+type: Evaluation Framework
+report_issue: https://github.com/deepset-ai/haystack-core-integrations/issues
+logo: /logos/deepeval.png
+version: Haystack 2.0
+toc: true
+---
+
+### Table of Contents
+
+- [Overview](#overview)
+- [Installation](#installation)
+- [Usage](#usage)
+    - [DeepEvalEvaluator](#DeepEvalEvaluator)
+
+## Overview
+
+[DeepEval](https://docs.confident-ai.com/) is an open source framework for model-based evaluation to evaluate your LLM applications by quantifying their performance on aspects such as correctness, tonality, hallucination, fluency, etc. More information can be found on the [documentation page](https://docs.haystack.deepset.ai/v2.0/docs/deepevalevaluator).
+
+## Installation
+
+Install the DeepEval integration:
+```bash
+pip install deepeval-haystack
+```
+
+## Usage
+
+Once installed, you will have access to a [DeepEvalEvaluator](https://docs.haystack.deepset.ai/v2.0/docs/deepevalevaluator) that supports a variety of model-based evaluation metrics: 
+- G-Eval
+- Summarization
+- Faithfulness
+- Answer Relevancy
+- Contextual Relevancy
+- Contextual Precision
+- Contextual Recall
+- Ragas
+- Hallucination
+- Toxicity
+- Bias
```

Comment:
We don't support all of those metrics, btw. Only a subset - https://github.com/deepset-ai/haystack-core-integrations/blob/d02286c714d9458f7e4a4038719cb17a8fcde27c/integrations/deepeval/src/haystack_integrations/components/evaluators/deepeval/metrics.py#L20

Reply:
Updated 👍